components/dashboards-web-component/src/gadgets-generation-wizard/components/GadgetsGenerationWizard.jsx

```diff
@@ -532,13 +532,22 @@ class GadgetsGenerationWizard extends Component {
         return (
             <MuiThemeProvider muiTheme={defaultTheme}>
                 <div>
-                    <Header title={<FormattedMessage id="widget-gen-wizard.title" defaultMessage="Widget Designer" />}/>
+                    <Header
+                        title={<FormattedMessage id="widget-gen-wizard.title" defaultMessage="Widget Designer" />} />
```

Comment:
Any reason for internationalizing only selected strings?

Reply:
This was the existing implementation, it was not changed. The highlighted change is the addition of a line break to prevent exceeding 120 character length.